Description: fix documentation privacy breaches
Author: Martin <debacle@debian.org>
Origin: vendor
Last-Update: 2018-03-10
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -370,8 +370,7 @@
 
 html_context = {
     'css_files': [
-        'https://media.readthedocs.org/css/sphinx_rtd_theme.css',
-        'https://media.readthedocs.org/css/readthedocs-doc-embed.css',
+        '_static/css/css/sphinx_rtd_theme.css',
         # '_static/css/custom.css',
     ],
 }
--- a/docs/adoptions.rst
+++ b/docs/adoptions.rst
@@ -16,32 +16,24 @@
 Debian
 ------
 
-.. image:: http://pyftpdlib.googlecode.com/svn-history/wiki/images/debian.png
-
 A `.deb packaged version of pyftpdlib <http://packages.debian.org/sid/python-pyftpdlib>`__
 is available.
 
 Fedora
 ------
 
-.. image:: http://pyftpdlib.googlecode.com/svn-history/wiki/images/fedora.png
-
 A `RPM packaged version <https://admin.fedoraproject.org/pkgdb/packages/name/pyftpdlib>`__
 is available.
 
 FreeBSD
 -------
 
-.. image:: http://pyftpdlib.googlecode.com/svn-history/wiki/images/freebsd.gif
-
 A `freshport <http://www.freshports.org/ftp/py-pyftpdlib>`__
 is available.
 
 GNU Darwin
 ----------
 
-.. image:: http://pyftpdlib.googlecode.com/svn-history/wiki/images/gnudarwin.png
-
 `GNU Darwin <http://www.gnu-darwin.org>`__ is a Unix distribution which focuses
 on the porting of free software to Darwin and Mac OS X. pyftpdlib has been
 recently included in the official repositories to make users can easily install
@@ -55,8 +47,6 @@
 Google Chrome
 -------------
 
-.. image:: http://pyftpdlib.googlecode.com/svn-history/wiki/images/chrome.jpg
-
 `Google Chrome <http://www.google.com/chrome>`__ is the new free and open
 source web browser developed by Google.
 `Google Chromium <http://code.google.com/intl/it-IT/chromium/>`__, the open
@@ -66,8 +56,6 @@
 Smartfile
 ---------
 
-.. image:: http://pyftpdlib.googlecode.com/svn-history/wiki/images/smartfile.jpg
-
 `Smartfile <http://www.smartfile.com>`__ is a market leader in FTP and online
 file storage that has a robust and easy-to-use web interface. We utilize
 pyftpdlib as the underpinnings of our FTP service. Pyftpdlib gives us the
@@ -76,8 +64,6 @@
 Bazaar
 ------
 
-.. image:: http://pyftpdlib.googlecode.com/svn-history/wiki/images/bazaar.jpg
-
 `Bazaar <http://bazaar-vcs.org/>`__ is a distributed version control system
 similar to Subversion which supports different protocols among which FTP.
 As for `Google Chrome <http://www.google.com/chrome>`__, Bazaar recently
@@ -86,8 +72,6 @@
 Python for OpenVMS
 ------------------
 
-.. image:: http://pyftpdlib.googlecode.com/svn-history/wiki/images/pyopenvms.png
-
 `OpenVMS <http://h71000.www7.hp.com/index.html?jumpid==/go/openvms>`__ is an
 operating system that runs on the `VAX <http://en.wikipedia.org/wiki/VAX>`__
 and `Alpha <http://en.wikipedia.org/wiki/DEC*Alpha>`__ families of computers,
@@ -102,8 +86,6 @@
 OpenERP
 -------
 
-.. image:: http://pyftpdlib.googlecode.com/svn-history/wiki/images/openerp.jpg
-
 `OpenERP <http://openerp.com>`__ is an Open Source enterprise management
 software.  It covers and integrates most enterprise needs and processes:
 accounting, hr, sales, crm, purchase, stock, production, services management,
@@ -113,8 +95,6 @@
 Plumi
 -----
 
-.. image:: http://pyftpdlib.googlecode.com/svn-history/wiki/images/plumi.jpg
-
 `Plumi <http://plumi.org/wiki>`__ is a video sharing Content Management System
 based on `Plone <http://plone.org>`__ that enables you to create your own
 sophisticated video sharing site.
@@ -124,8 +104,6 @@
 put.io FTP connector
 --------------------
 
-.. image:: http://pyftpdlib.googlecode.com/svn-history/wiki/images/putio.png
-
 A proof of concept FTP server that proxies FTP clients requests to
 `putio <http://put.io/>`__ via HTTP, or in other words an FTP interface to
 put.io Put.io is a storage service that fetches media files remotely and lets
@@ -137,8 +115,6 @@
 Rackspace Cloud's FTP
 ---------------------
 
-.. image:: http://pyftpdlib.googlecode.com/svn-history/wiki/images/rackspace-cloud-hosting.jpg
-
 `ftp-cloudfs <http://github.com/chmouel/ftp-cloudfs>`__ is a ftp server acting
 as a proxy to Rackspace `Cloud Files <http://www.rackspacecloud.com>`__. It
 allows you to connect via any FTP client to do upload/download or create
@@ -147,8 +123,6 @@
 Far Manager
 -----------
 
-.. image:: http://pyftpdlib.googlecode.com/svn-history/wiki/images/farmanager.png
-
 `Far Manager <http://farmanager.com/>`__ is a program for managing files and
 archives in Windows operating systems.
 Far Manager recently included pyftpdlib as a plug-in for making the current
@@ -158,8 +132,6 @@
 Google Pages FTPd
 -----------------
 
-.. image:: http://pyftpdlib.googlecode.com/svn-history/wiki/images/google-pages.gif
-
 `gpftpd <http://arkadiusz-wahlig.blogspot.com/2008/04/hosting-files-on-google.html>`__
 is a pyftpdlib based FTP server you can connect to using your Google e-mail
 account.
@@ -170,8 +142,6 @@
 Peerscape
 ---------
 
-.. image:: http://pyftpdlib.googlecode.com/svn-history/wiki/images/peerscape.gif
-
 `Peerscape <http://www.peerscape.org/>`__ is an experimental peer-to-peer social
 network implemented as an extension to the Firefox web browser. It implements a
 kind of serverless read-write web supporting third-party AJAX application
@@ -191,8 +161,6 @@
 Symbian Python FTP server
 -------------------------
 
-.. image:: http://pyftpdlib.googlecode.com/svn-history/wiki/images/symbianftp.png
-
 An FTP server for Symbian OS: http://code.google.com/p/sypftp/
 
 ftp-cloudfs
@@ -321,20 +289,14 @@
 www.bitsontherun.com
 --------------------
 
-.. image:: http://pyftpdlib.googlecode.com/svn-history/wiki/images/bitsontherun.png
-
 http://www.bitsontherun.com
 
 www.adcast.tv
 -------------
 
-.. image:: http://pyftpdlib.googlecode.com/svn-history/wiki/images/adcast.png
-
 http://www.adcast.tv http://www.adcast.tv
 
 www.netplay.it
 --------------
 
-.. image:: http://pyftpdlib.googlecode.com/svn/wiki/images/netplay.jpg
-
-http://netplay.it/
\ No newline at end of file
+http://netplay.it/
